apps/webstore/src/app/pages/home/home.component.html

```diff
@@ -77,7 +77,13 @@ <h2 class="discriptive-title title-font" i18n="@@home-main-text-title">
                         ><img class="so-me-icon" src="/assets/logos/soMe/instagram.svg" alt="Treecreate instagram side"
                     /></a>
                 </div>
-                <button class="tc-button-50" i18n="@@create-my-family-tree">Create my family tree</button>
+                <button
+                    class="tc-button-50 purhcase-button"
```

Comment:
You have a little typo in the CSS class name (needs to be corrected in the CSS file as well)

```suggestion
                    class="tc-button-50 purchase-button"
```